File: programmers/Level1/3진법뒤집기.java
class Solution {
    public int solution(int n) {
        String tmp = "";
        while(n > 0) {
            tmp = String.valueOf(n % 3) + tmp;
            n /= 3;
        }

        int answer = 0;
        for(int i = 0; i < tmp.length(); i++) {
            if(tmp.charAt(i) != '0') {
                answer += (tmp.charAt(i) - '0') * Math.pow(3, i);
            }
        }
        return answer;
    }
}

File: programmers/Level1/3진법뒤집기.py
def solution(n):
    answer = ''
    
    while n > 0:
        n, mod = divmod(n, 3)  
        answer += str(mod)

    return int(answer, 3)

# divmod(x, y) : x를 y로 나눈 몫과 나머지를 반환
# int(x, base) : base 진법으로 구성된 str 형식의 수를 10진법으로 변환